src/context-providers/builder/hook.ts

```diff
@@ -169,6 +169,28 @@ export const useBuilder = () => {
         });
     }, []);
 
+    const updateElementSchema = useCallback(
+        (elements?: TElementMap, orderedIdentifiers?: IElementIdentifier[]) => {
+            if (orderedIdentifiers) {
+                dispatch({
+                    type: "update-schema-element",
+                    payload: {
+                        elements: elements,
+                        orderedIdentifiers: orderedIdentifiers,
+                    },
+                });
+            } else {
+                dispatch({
+                    type: "update-schema-element",
+                    payload: {
+                        elements: elements,
+                    },
+                });
+            }
```

Comment:
- is it necessary to exclude `orderedIdentifiers` if it is not defined?
- use shorthand if key and variable name are the same